### PR TITLE
TG-131: Added Input to shared components

### PR DESCRIPTION
**Pull request by FefeTroanes**
Here there is the Input shared component, it has the following props:
**_1. type:_** Where you can choose the type input (text, date, number and so on).
**_2. htmlForPropAndLabel:_** Where the htmlFor for accessibility is specified. It's has also de `Label` tag content
**_3. register:_** This is where you send the register of the Forms as a prop
**_4. name:_** Here you specify the `{...register(name)}` and the content for the `name` tag
**_5. error:_** The error that should show if it is needed
 **_6. width:_** The width of the input

It has **_NO_** functionality, it's just the component
(I tested it on the employee's tab and it worked fine)

### CODE REVIEW

File: src/Components/Shared/Input/index.jsx [hunk 1]
```diff
@@ -0,0 +1,14 @@
+import React from 'react';
+import styles from './input.module.css';
+
+const Input = ({ type, htmlForProp, label, register, name, error, width }) => {
+  return (
+    <div>
+      <label htmlFor={htmlForProp}>{label}</label>
+      <input style={{ width }} type={type} {...register(name)} />
```
Comment (nicolascostanza) [inline src/Components/Shared/Input/index.jsx:8]: Aca el register funciona ? es pregunta. Porque lo veo como prop y despues hardcodeado

Comment (javierlazzarini) [inline src/Components/Shared/Input/index.jsx:8]: al imput no le faltaria el atributo name ?
podria usarse la misma prop en htmlfor , name y register

Comment (FefeTroanes) [reply src/Components/Shared/Input/index.jsx:8]: La misma prop para htmlFor y el contenido de la label puede ser. El atributo name creeria que no hace falta, yo lo probe asi como esta con los employees y me funcionaba

Comment (FefeTroanes) [reply src/Components/Shared/Input/index.jsx]: Igual se la agregue

**Review by martinalbrnz — APPROVED** [review]
LGTM!

**Review by nicolascostanza — APPROVED** [review]

**Pull request merged**